pkg/client/unversioned/cache/fifo_test.go

```diff
@@ -107,7 +107,7 @@ func TestFIFO_addUpdate(t *testing.T) {
 func TestFIFO_addReplace(t *testing.T) {
 	f := NewFIFO(testFifoObjectKeyFunc)
 	f.Add(mkFifoObj("foo", 10))
-	f.Replace([]interface{}{mkFifoObj("foo", 15)})
+	f.Replace([]interface{}{mkFifoObj("foo", 15)}, "0")
```

Comment:
Nit: Real data probably would have rv being >= the highest RV in the list. So if that 15 is supposed to be a resource version, then the "0" should probably be at least "15"...